R/setops.R

```diff
@@ -6,7 +6,7 @@ setdiff_ = function(x, y, by.x=seq_along(x), by.y=seq_along(y), use.names=FALSE)
   by.x = colnamesInt(x, by.x, check_dups=TRUE)
   if (!nrow(y)) return(unique(x, by=by.x))
   by.y = colnamesInt(y, by.y, check_dups=TRUE)
-  if (length(by.x) != length(by.y)) stopf("length(by.x) != length(by.y)")
+  if (length(by.x) != length(by.y)) stopf("length(by.x) != length(by.y)") # nontranslate
```

Comment:
Hmm, I _think_ this approach should be better. I am forgetting whether this keeps the string out of `po_extract()`. Either way we shouldn't attempt lookup at run time.

```suggestion
  if (length(by.x) != length(by.y)) stopf("length(by.x) != length(by.y)", domain=NA)
```